### DISCUSSION TITLE
No Breakpoints with Visual Studio 17.10.x for assemblies dependent on Avalonia

### DISCUSSION THREAD

**Discussion by LaurentInSeattle** [AvaloniaUI/Avalonia#15999] [Q&A]
Hi all, 

I am using Microsoft Visual Studio Community 2022 (64-bit) Version 17.10.1 for my current Avalonia mini-project, which is NOT using AOT, at least for now. It's using the official Avalonia nugets in version 11 along with .Net 8.

Since VS version 17.10.0 it is not possible to have breakpoints being hit in all assemblies that have a dependency on Avalonia.

Changing project settings to use compiled bindings or not does not change anything.

Same C# code compiled on a Mac with Jet Brains Rider (Free trial) works fine and all breakpoints get hit, Avalonia dependent or not.

On the PC+VS, adding the line Debugger.Break() ; causes the debugger to break, and then stepping and inspecting do work as expected.

Any idea ? or workaround ? 

Thanks!
L.

**Comment by LaurentInSeattle** [comment]
And still same problem with VS 17.10.02 released earlier today (June 11th)

**Comment by LaurentInSeattle** [comment]
Any comments or suggestions ??? 
Am I the only one having this problem ? 

This is a big issue here...

**Reply by timunie** [reply]
If Rider works and VS not, well that's not really something we can control. If you have  minimal sample one may try over here but well that's probably a hard thing for us to digg into
